README.md

```diff
@@ -5,3 +5,35 @@
 > The Chinese Graves Project is a digital humanities initiative based at Stanford that will build and harness an interactive spatial and textual analysis platform to examine the phenomenon of grave relocation in modern China, a campaign that has led to the exhumation and reburial of 10 million corpses in the past decade alone, and has transformed China’s graveyards into sites of acute personal, social, political, and economic contestation.
 
 &mdash; Tom Mullaney, "[Introducing the Chinese Graves Project: A New Digital Humanities Initiative at Stanford University](http://tsmullaney.com/?p=412)"
+
+## Install for Development
```

Comment:
Can you make a note specifying the node and npm versions?

node v0.10.36
npm v3.7.5